pkg/engine/task/task_apply.go

```diff
@@ -145,3 +237,77 @@ func isHealthy(ro []runtime.Object) error {
 	}
 	return nil
 }
+
+// copy from k8s.io/kubectl@v0.16.6/pkg/util/apply.go, but with different annotation
+// GetOriginalConfiguration retrieves the original configuration of the object
+// from the annotation, or nil if no annotation was found.
+func getOriginalConfiguration(obj runtime.Object) ([]byte, error) {
+	annots, err := metadataAccessor.Annotations(obj)
+	if err != nil {
+		return nil, err
+	}
+
+	if annots == nil {
+		return nil, nil
+	}
+
+	original, ok := annots[kudo.LastAppliedConfigAnnotation]
```

Comment:
I got stuck at this line. Are we backwards compatible after this is merged? In a scenario where someone re-runs a `deploy` plan for an existing operator (after updating KUDO), all plan resources will be `patch`ed, correct? However, there is no `LastAppliedConfigAnnotation` so it will `return nill, nil` and the subsequent merges will... fail?

Reply:
Yeah, this is a problem. I'll try to figure something out about this...